packages/imperative/src/imperative/src/OverridesLoader.ts

```diff
@@ -86,7 +86,27 @@ export class OverridesLoader {
                 Manager = resolve(resolvePath, "../", overrides.CredentialManager);
             }
 
-            await CredentialManagerFactory.initialize({
+            // Load credential manager options from team config if available
```

Comment:
Minor detail: Unless I badly misunderstand, the comment about loading from "team config" is a typo.